package/contents/ui/UpcomingEvents.qml

```diff
@@ -227,6 +227,8 @@ CalendarManager {
 
 	Connections {
 		target: timeModel
-		onMinuteChanged: upcomingEvents.tick()
+		function onMinuteChanged() {
+		 upcomingEvents.tick()
```

Comment:
indentation is off here

Reply:
Thanks, fixed but the project looks dead.